conda/recipes/librmm/meta.yaml

```diff
@@ -18,10 +18,17 @@ requirements:
     - ninja
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }} {{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} {{ cuda_version }}
+    {% else %}
+    - cuda-version {{ cuda_version }}
+    - {{ compiler('cuda') }}
+    {% endif %}
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
+    {% if cuda_major == "11" %}
     - cudatoolkit ={{ cuda_version }}
```

Comment:
I'm curious, what does rmm require from the CTK? It seems a bit suspect that we need the CTK for CUDA 11 but don't need anything at all for CUDA 12. Is this requirement actually superfluous, or am I missing something obvious? If this is just to pin a version, should we include `cuda-version` in the host reqs (unconditionally since it works for CUDA 11 and CUDA 12)?

Reply:
Added a suggestion below to use `cuda-version`

Reply:
Let’s defer on changing CUDA 11 packaging. I will leave this open to investigate in a future PR.

Reply:
Looks like `cuda-version` was added

The `cudatoolkit` question remains. AIUI RMM links to `libcudart`, which `cudatoolkit` provides. Whether we should require installation of `cudatoolkit` when just using the headers though is a good question. Maybe we can convert this to an issue for further discussion?

Reply:
Looks like `cudatoolkit` was dropped from `run`. Marking resolved